zach_brady/class6/.ruby-version

```diff
@@ -0,0 +1 @@
+system
```

Comment:
@zzzbra you probably don't need this any more now that you have your machine re-set up